module/config.mjs

```diff
@@ -897,6 +897,22 @@ preLocalize("featureTypes.class.subtypes", { sort: true });
 
 /* -------------------------------------------- */
 
+/**
+ * The various properties of a feature item.
+ * @enum {object}
+ */
+DND5E.featureProperties = {
+  concentration: {
+    label: "DND5E.FeaturePropertyCon"
+  },
+  mgc: {
+    label: "DND5E.FeaturePropertyMgc"
+  }
+};
+preLocalize("featureProperties", {keys: ["label"]});
```

Comment:
I feel like this should just be a temporary solution. Maybe we should look into having a single `DND5E.itemProperties` object, and each property having an `itemTypes` array to show which items it shows up on. I looked into it only briefly, but ammo makes it more complicated, since it should depend on both item *type* (consumable) but then also item *subtype* (ammo).